root/static/scripts/common/entity.js

```diff
@@ -231,8 +231,14 @@ import formatTrackLength from './utility/formatTrackLength';
 
     class Label extends CoreEntity {
         selectionMessage() {
+            let code = this.toJSON().label_code;
```

Comment:
I don't think you need toJSON() here? Also, I'd rather this was a const and we had a different variable for the code. This variable can already start as the empty string, that way you don't need the `else`.

```suggestion
            const code = this.label_code;
            let codeSection = '';            
```

Reply:
Indeed, no need for the toJSON